nextflow_schema.json

```diff
@@ -107,35 +107,6 @@
             },
             "description": "all the trim option are listed below"
         },
-        "hmmer_search_options": {
-            "title": "HMMER search options",
-            "type": "object",
-            "description": "Options for HMMER search",
-            "default": "",
-            "properties": {
-                "hmmdir": {
-                    "type": "string",
-                    "description": "Directory with hmm files which will be searched for among ORFs",
-                    "pattern": "^\\S+",
-                    "format": "directory-path",
-                    "fa_icon": "fas fa-folder-open"
-                },
-                "hmmpattern": {
-                    "type": "string",
-                    "default": "*.hmm",
-                    "description": "specify which pattern hmm files end with",
-                    "fa_icon": "fas fa-barcode"
-                },
```

Comment:
Where did this go, don't we use it?

Reply:
Now I see, you put it among the hidden options. Is that how similar options are handled by other pipelines?

Reply:
I think that it is a mistake, it should be visible